admin_guide/managing_networking.adoc

```diff
@@ -1207,18 +1207,16 @@ objects:
 
 HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
 requests that traffic to the host be HTTPS. This is useful for ensuring secure
-interactions with websites, or to offer secure application for the user's
+interactions with websites, or to offer a secure application for the user's
 benefit.
 
-The default behavior is to drop any HTTP requests that require HTTPS. Enabling
-HSTS lets you set up a redirect so that the first time a client requests the
-target it is redirected to HTTPS. Any subsequent attempts will get a HTTPS
-header in the response, meaning future requests will be HTTPS.
-
-HSTS adds a Strict Transport Security header to HTTPS responses from the site.
-The client (often a browser) reads the header and, from then on, can enforce the
-policy. All requests to the HTTP URL are changed to HTTPS before the request is
-sent, eliminating the need for a redirect.
+The {product-title} default behavior is to drop any HTTP requests that require
```

Comment:
Actually, this first sentence doesn't really make sense either.  That behavior is controlled by a separate setting.